2018/index.html

```diff
@@ -114,7 +114,7 @@ <h2 class="home-subhead">Featured Speakers</h2>
         
         <li>
           
-<a class="speaker" data-key="addaleax">
+<a class="speaker" href="speakers.html#speaker-addaleax">
```

Comment:
Could we delete this `data-key` ??

Reply:
Yes, i updated the overlay system to accomodate for it.